oximeter/oxql-types/src/point.rs

```diff
@@ -169,10 +169,10 @@ impl CumulativeDatum {
 /// A single list of values, for one dimension of a timeseries.
 #[derive(Clone, Debug, Deserialize, JsonSchema, PartialEq, Serialize)]
 pub struct Values {
-    // The data values.
-    pub(super) values: ValueArray,
-    // The type of this metric.
-    pub(super) metric_type: MetricType,
+    /// The data values.
+    pub values: ValueArray,
+    /// The type of this metric.
+    pub metric_type: MetricType,
```

Comment:
> Interesting, I wonder why this changed?

Ah I added doc comments here which caused that change to happen. There are no semantic changes to the OpenAPI document.